src/strategies/keystore.rs

```diff
@@ -95,6 +95,10 @@ impl KeyStore {
     pub fn len(&self) -> usize {
         self.keys.len()
     }
+
+    pub fn is_empty(&self) -> bool {
```

Comment:
`pub fn len()` and `pub fn is_empty()` often go hand-in-hand to provide the same interface as std types